apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.html

```diff
@@ -27,74 +27,57 @@
-                                            (click)="changeBranch(company.uniqueName, branch?.uniqueName, $event)">
-                                            <i class="icon-branch-connect mr-2"></i>
-                                            <span class="d-flex flex-column">
-                                                <span class="branch-alias">{{ branch.alias }}</span>
-                                                <span class="font-12 branch-company-name">{{ company.name }}</span>
-                                            </span>
-                                        </a>
-                                    </li>
-                                </ng-container>
-                                <li class="show-more-branches font-12" *ngIf="company.branches?.length > 5"
-                                    (click)="showAllBranches(company)">{{localeData?.show_more}}</li>
-                            </ul>
                         </li>
                     </ul>
                     <giddh-page-loader *ngIf="(isCompanyRefreshInProcess$ | async)"></giddh-page-loader>
                 </div>
             </tab>
-            <tab [heading]="commonLocaleData?.app_branch" [customClass]="'icon-branch1'"
-                [disabled]="isBranchTabDisabled" (selectTab)="tabChanged('branch')">
+            <tab [heading]="commonLocaleData?.app_branch" [customClass]="'icon-branch1'" (selectTab)="tabChanged('branch')">
                 <div>
                     <p class="font-12 d-flex justify-content-between mt-2">
                         <span>{{branchList?.length}} {{localeData?.branches}}</span>
-                        <span class="digit-blue cursor-pointer"
-                            (click)="getCompanyBranches(1, companyBranches, true)">{{localeData?.refresh_manually}}</span>
+                        <span class="cursor-pointer" [tooltip]="localeData?.refresh_manually"
+                            (click)="getCompanyBranches(1, companyBranches, true)">
+                            <i class="icon-reload font-22"></i>
+                        </span>
                     </p>
-                    <p class="font-14 mt-2 digit-orange cursor-pointer" [routerLink]="['/pages/settings/create-branch']"
-                        *ngIf="activeCompany?.uniqueName === companyBranches?.uniqueName">{{localeData?.create_branch}}
+                    <p class="font-14 mt-2 digit-blue cursor-pointer" [routerLink]="['/pages/settings/create-branch']"
+                        *ngIf="activeCompany?.uniqueName === companyBranches?.uniqueName"><i class="icon-plus font-12"></i>
+                        <span class="digit-blue">{{localeData?.create_branch}}</span>
                     </p>
                 </div>
                 <div class="company-list branch-list mt-2">
                     <ul class="list-style-none font-14" *ngIf="!branchRefreshInProcess">
+                        <li *ngIf="showGoToCompany()">
```

Comment:
showGoToCompany should be a variable to enable caching in Angular and avoid multiple calculations